sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/MergeRowsExec.scala

```diff
@@ -45,6 +47,14 @@ case class MergeRowsExec(
     output: Seq[Attribute],
     child: SparkPlan) extends UnaryExecNode {
 
+  override lazy val metrics: Map[String, SQLMetric] = Map(
+    "numTargetRowsCopied" -> SQLMetrics.createMetric(sparkContext,
+      "Number of target rows rewritten unmodified because they did not meet any condition."),
+    "numTargetRowsUnused" -> SQLMetrics.createMetric(sparkContext,
```

Comment:
Discussed with @szehon-ho on unused metrics offline and decide to hold off adding them for now.